jdaviz/configs/default/plugins/export/export.py

```diff
@@ -389,6 +406,38 @@ def save_movie(self, viewer, filename, filetype, i_start=None, i_end=None, fps=N
 
         return filename
 
+    def save_subset_as_region(self, selected_subset_label, filename):
+        """
+        Save a subset to file as a `Region` object in the working directory.
+        Currently only enabled for non-composite spatial subsets. Can be saved
+        as a `.fits` or `.reg` file. If link type is currently set to 'pixel',
+        then a pixel region will be saved. If link type is 'wcs', then a sky
+        region will be saved. If a file with the same name already exists in the
+        working directory, it will be overwriten.
+        """
+
+        # type of region saved depends on link type
+        link_type = getattr(self.app, '_link_type', None)
+
+        region = self.app.get_subsets(subset_name=selected_subset_label,
+                                      include_sky_region=link_type == 'wcs')
+
+        if len(region) > 1:
+            raise NotImplementedError("Export not yet supported for composite subsets.")

```

Comment:
we should also make sure to show this in the UI similar to how is done in cubeviz spectral extraction.

Reply:
I added an overlay when you try to export a composite subset, but I can't get the placement of it correct. How does it look to you? 

If we will eventually add support for this, does the snackbar suffice?

Reply:
my suggestion would be to not use an overlay, but instead use the pattern in spectral extraction (and other places):
* remove the spatial filter from the subset component
* create a string traitlet in the plugin for `subset_invalid_msg` (or similar)
* observe `subset_selected` and determine if `subset.selected_obj` is valid for exporting or not.  If so, set the string to empty, if not set it to the message you want displayed ("exporting spectral subsets not yet supported" or "exporting composite subsets not supported")
* in `export()` check if this string is set, and if so, raise the error
* in vue, if the string is not empty, display it as red text:

```
<v-row v-if="subset_invalid_msg.length > 0">
  <span class="v-messages v-messages__message text--secondary">
            {{subset_invalid_msg}}
  </span>
</v-row>
```

* in vue, if the string is not empty, disable the export button:

```
<plugin-add-results
  ...
  :action_disabled="... || subset_invalid_msg.length > 0"
  ...
></plugin-add-results>
```